src/Memed/Soluti/Config.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Memed\Soluti;
+
+class Config
+{
+    /**
+     * @var array
+     */
+    protected $data = [];
+
+    /**
+     * Constructor.
+     */
+    public function __construct(array $data)
+    {
+        $this->data = $data;
+    }
+
+    /**
+     * Retrieves url of cess service.
+     */
+    public function cessUrl(): ?string
+    {
+        return $this->data['url_cess'];
```

Comment:
**nip**:
Since we don't have validation on `$data`, maybe we could change here to `return $this->data['url_cess'] ?: '';`

Reply:
I don't think so. We don't have any validations on this package at all and signature service doesn't work without `url_cess`. But I agree, we need to put some validations on it soon.